cloudformation-templates/pattern-1/pattern-1-with-puppet-cloudformation.template.yml

```diff
@@ -60,6 +60,10 @@ Mappings:
 # User inputs
 #############################
 Parameters:
+  AMI:
+    Type: String
+    Default: USE_DEFAULT
```

Comment:
What would be the reason for having the value "USE_DEFAULT" in the Default property? Wouldn't it be more meaningful to have the default AMI ID here?

Reply:
Since the script already has the Ubuntu AMIs mapped with their regions, by passing "USE_DEFAULT", the user can let script to select and use relevant Ubuntu AMI.

Reply:
Changed the term to "READ_FROM_REGIONMAP" to make it more clear.